src/services/tree-sitter/__tests__/languageParser.test.ts

```diff
@@ -1,118 +1,106 @@
+// npx jest src/services/tree-sitter/__tests__/languageParser.test.ts
```

Comment:
This test was mocking too much, and isn't compatible with the latest version of WASM tree-sitter. I updated it appropriately.